.idea/Cartoon-Database.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you delete the .idea folder from github please? delete it locally and add it to a .gitignore - also add your .DS_Store file